modules/ossm-cert-manager-istio-csr-revisionbased-strategy.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/install/ossm-cert-manager-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="revision-based-istio-csr-installation_{context}"]
+= istio-csr installation: RevisionBased update strategy
+
+If you plan to set the `spec.updateStrategy` field to `RevisionBased` when you create and install your `Istio` resource, follow this procedure.
+
+.Procedure
+
+. Specify all the Istio revisions to your `istio-csr` deployment.
+
+. Add the jetstack charts to your local Helm repo by running the following command:
```

Comment:
See the comment above. In the upstream cert-manager docs, Jetstack is a proper noun. Also s/repository/repo.